jpo-ode-plugins/src/test/java/us/dot/its/jpo/ode/plugin/j2735/builders/VehicleSizeBuilderTest.java

```diff
@@ -59,7 +59,7 @@
    @Test
    public void shouldReturnUndefinedVehicleWidth() {
```

Comment:
Since we've changed this to return 0 instead of null, should we change the test names to correspond? "shouldReturnZeroVehicleWidth"?

Reply:
That seems reasonable

Reply:
Addressed